### ISSUE TITLE
error while parsing long[] as a paramenter

### ISSUE THREAD

**Issue by bjin** [lovrop/topcoder-moj#1]
in today's SRM(SRM 527), the hard problem contains long[] as input, moj generated the following codes:

```
long[] values             = {1, 1000000000, 1000000000000000000};
```

which caused compile error with jdk1.7

**Comment by lovrop** [comment]
Thanks for the report. I'll take a look and bring the test coverage of Java support up to speed with C++.

**Comment by lovrop** [comment]
Fixed in v4.17.
